EIPS/eip-778.md

```diff
@@ -74,12 +80,14 @@ Records are signed and encoded as follows:
 ### Text Encoding
 
 The textual form of a node record is the base64 encoding of its RLP representation,
-prefixed by `enr:`.
+prefixed by `enr:`. Implementations should use the [URL-safe base64 alphabet][base64url]
```

Comment:
"should" or "must"?

Reply:
I meant "must", but it sounds silly.

Reply:
Ah, I see that many (but not all) EIPs explicitly use [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt)'s definitions: https://github.com/ethereum/EIPs/search?q=2119&type=Code

It might sound silly, but for the sake of precision, I would recommend using RFC 2119 words here. Ambiguity about "is this preferred or required" can lead to some sloppy real-world implementations, and wasted time disagreeing about whether code is compliant.